src/substrait/__init__.py

```diff
@@ -2,3 +2,7 @@
     from ._version import __version__
 except ImportError:
     pass
+
+__substrait_version__ = "0.30.0"
+__substrait_hash__ = "3259a1ba"
+__minimum_substrait_version__ = "0.30.0"
```

Comment:
What does the "minimum" version mean exactly? (is that a concept of substrait itself?)

Reply:
Looking forward, it's supposed to represent the oldest version of substrait that this library can parse a plan from.  I don't think it will be used right away.